packages/peregrine/lib/talons/CartPage/ProductListing/EditModal/__tests__/useProductForm.spec.js

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import { act } from 'react-test-renderer';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+
+import { useAppContext } from '../../../../../context/app';
+import createTestInstance from '../../../../../util/createTestInstance';
+import { useProductForm } from '../useProductForm';
+
+jest.mock('@apollo/react-hooks', () => ({
+    useMutation: jest
+        .fn()
+        .mockReturnValue([jest.fn(), { called: false, loading: false }]),
+    useQuery: jest.fn().mockReturnValue({
+        data: null,
+        error: null,
+        loading: true
+    })
+}));
+
+jest.mock('../../../../../context/app', () => {
+    const state = {};
+    const api = {
+        closeDrawer: jest.fn()
+    };
+    const useAppContext = jest.fn(() => [state, api]);
+
+    return { useAppContext };
+});
+
+jest.mock('../../../../../context/cart', () => {
+    const state = {
+        cartId: 'cart123'
+    };
+    const api = {};
+    const useCartContext = jest.fn(() => [state, api]);
+
+    return { useCartContext };
+});
+
+const Component = props => {
+    const talonProps = useProductForm(props);
+    return <i talonProps={talonProps} />;
+};
+
+const configItemResponse = {
```

Comment:
What do you think about moving this to `__fixtures__/configResponses.js`?

Reply:
Moved in ec2be48